frontend/Components/Header/adminlink/index.js

```diff
@@ -0,0 +1,15 @@
+import React, { Component } from 'react';
+import { Link } from 'react-router';
+import styles from './styles';
+
+const AdminLink = () => {
+  return (
+    <div className={styles.adminPageContainer}>
```

Comment:
I don't think we need a whole new class for the admin link, it makes the code too spread out I think. Can we integrate the link directly into "Header" or "NavigationBar"? Otherwise looks good!

Reply:
I integrated it into UserMenu, is that okay?

Reply:
![WX20210211-142509](https://user-images.githubusercontent.com/46878322/107607726-ffe2ec80-6c74-11eb-898a-299197bed432.png)